larsim/IonizationScintillation/IonAndScint_module.cc

```diff
@@ -123,10 +129,7 @@ namespace larg4
     {
         std::cout << "IonAndScint endJob." << std::endl;
 
-        if(fISAlg)
-        {
-            delete fISAlg;
-        }
+        if(fISAlg) delete fISAlg;
```

Comment:
See comment above about using `std::unique_ptr<ISCalc>` instead as the type of `fISAlg`.